docs/device_gateway.md

```diff
@@ -27,8 +27,8 @@ If you are interested in more details about how to connect your devices and sens
 
 At present, the following IoT Agents and supported IoT protocols in the IoT Platform are:
 
-- [Ultralight 2.0](https://github.com/telefonicaid/iotagent-ul): SensorML based simple text protocol.
-- [JSON](https://github.com/telefonicaid/iotagent-json): a JSON based key-value pair protocol.
+- [Ultralight 2.0](https://github.com/telefonicaid/iotagent-ul/tree/release/1.1.0): SensorML based simple text protocol.
```

Comment:
I think it should be better to link to particular versions (tags).

For example, see PR #30